ui/src/message_popup/data_lists.jsx

```diff
@@ -0,0 +1,142 @@
+    ell: null,
+    learningDisabilities: 'ADHD',
+    academicPerformance: null,
+    interests: 'Wants to be a firefighter',
+    familyBackground: 'Divorced parents, younger brother',
+    ses: 'Free and reduced lunch'
+  },
+  {
+    id: 3,
+    name: 'Maia',
+    grade: '7th grade',
+    gender: 'female',
+    race: 'Caucasian',
+    behavior: 'Discipline report',
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'In theater and loves art',
+    familyBackground: 'Single mom, two olders sisters',
+    ses: null
+  },
+  { 
+    id: 4, 
+    name: 'Hayin',
+    grade: '7th grade',
+    gender: 'female',
+    race: 'Korean',
+    behavior: null,
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Plays the flute, very involved in church, loves coding',
+    familyBackground: '1st generation parents, twin sister',
+    ses: 'Free and reduced lunch'
+    
+  },
+  {
+    id: 5,
+    name: 'Mike',
+    grade: '7th grade',
+    gender: 'male',
+    race: 'African American',
+    behavior: null,
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Plays in a band, sings in the school chorus',
+    familyBackground: '1 older brother',
+    ses: null,
+  },
+  {
+    id: 6,
+    name: 'Jasmine',
+    grade: '7th grade',
+    gender: 'female',
+    race: 'African American',
+    behavior: 'Quiet',
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Plays tennis and soccer',
+    familyBackground: 'Divorced parents, only child',
+    ses: 'Free and reduced lunch'
+  },
+  {
+    id: 7,
+    name: 'Miquel',
+    grade: '7th grade',
+    gender: 'male',
+    race: 'Latino',
+    behavior: 'Attendance issues',
+    ell: 'Yes, Spanish is native language',
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Has an after school job',
+    familyBackground: '3 younger siblings',
+    ses: 'Free and reduced lunch'
+  },
+  {
+    id: 8,
+    name: 'Ada',
+    grade: '7th grade',
+    gender: 'female',
+    race: 'Haitian',
+    behavior: null,
+    ell: 'Yes, Haitian Creole is native language',
+    learningDisabilities: 'Auditory disability',
+    academicPerformance: null,
+    interests: 'High-achiever, teacher\'s pet. Wants to be a doctor, is in yearbook',
+    familyBackground: 'Aunt is guardian',
+    ses: null
+  },
+  {
+    id: 9,
+    name: 'Steve',
+    grade: '7th grade',
+    gender: 'male',
+    race: 'Caucasian',
+    behavior: 'Often tired in class',
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Prefers ELA over math and science. Plays drums at home with his family. Has an after school job',
+    familyBackground: 'Younger sister',
+    ses: null
+  },
+  { 
+    id: 10, 
+    name: 'Sasha',
+    grade: '7th grade',
+    gender: 'female',
+    race: '?'
```

Comment:
Let's give her a race so it feels authentic (given that there are all sorts of questions about presenting these profiles in a way that avoids people defaulting to stereotypes, and we'll have to improve that for sure).  I rolled on http://neilcic.com/identity.html and Sasha is now African American. :)